File: C/Ex13.c
#include<stdio.h>
#include<stdlib.h>

//Função principal do programa
int main (int argc, char const *argv[])

//PROGRAMA 13 - ÁREA DO RETÂNGULO >=100 mensagem TERRENO GRANDE, caso contrário TERRENO PEQUENO

{
//Declaração das variáveis
int b, h, area;

// Exibe na tela 
printf("Calculo da area do retangulo");

//Quebra de linha
printf("\n");

//Solicita entrada de dados para o usuário e grava o valor digitado na variável "b"
printf("Digite o valor da base: ");
scanf("%i", &b);

//Solicita entrada de dados para o usuário e grava o valor digitado na variável "h"
printf("Digite o valor da altura: ");
scanf("%i", &h);

//Faz a multiplicação dos valores obtidos acima
area = b * h;
//Exibe o valor da area
printf("O valor da area e: %i",area);

if(area >=100){
  printf("\nTerreno grande!"); 
}
else {
    printf("\nTerreno pequeno!");
}
return 0;
}

File: PYTHON/Ex13.py
# Se a área do retângulo for maior ou igual a 100, exibir Terreno Grande! caso contrário Terreno Pequeno!
print("Cálculo da área do retângulo")

b = int(input('Digite o valor da base: '))
h = int(input('Digite o valor da altura: '))

area = b * h
print(f"O valor da área é: {area}")

if area >= 100:
 print(f"Terreno grande!")
else:
 print(f"Terreno Pequeno!")